tests/test_wave_01.py

```diff
@@ -143,12 +144,13 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
+    assert updated_data["watched"] == [FANTASY_2,HORROR_1]
```

Comment:
This definitely works! You could also use `in` to make the assertions and capture more conditions like:
```python
    assert updated_data["watched"][1] == movie_to_watch
    assert movie_to_watch in updated_data["watched"]
    assert FANTASY_2 in updated_data["watched"]
    assert FANTASY_1 in updated_data["watchlist"]
    assert movie_to_watch not in updated_data["watchlist"]
```